platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/LogoutRequestService.java

```diff
@@ -180,10 +179,15 @@ public Response sendLogoutRequest(@QueryParam("EncryptedNameIdTime") String encr
           return buildLogoutResponse(msg);
         }
 
+        SecurityToken IdpSecToken = getIdpSecurityToken();
```

Comment:
✏️ Best practice is for the first letter to be lower-case, so it should be `idpSecToken`. (this also applies to the other 2 similar usages below in this file)